### PR TITLE
docs(releases): Add definition of Experiments to Releases page

### PR DESCRIPTION
**Pull request by akshayabd**

### CODE REVIEW

File: _posts/releases/1300-09-02-early-release-beta-GA.md [hunk 1]
```diff
@@ -1,18 +1,24 @@
...
-What is the definition for releases that are Early Release, Beta, and GA?
+What is the definition for releases that are Experiments, Early Release, Beta, and GA?
 <br><br>
 ### Answer
 This is a description that varies from company to company. We decided to publish what each one of these release definitions are for Armory, and where we suggest you install them. Regardless of what stage a feature is in, we would love to hear some feedback about it.
 <br><br>
+### Experiment
+An experiment is a test version of a new feature that we think will solve some of your pain points. This is your opportunity to provide us feedback. We want to hear the good, the bad, and the ugly, so we can create the best feature to solve your problem.
+<br><br>
+**Where To Install Early Release**
```
Comment (brian-armory) [inline _posts/releases/1300-09-02-early-release-beta-GA.md]: Shouldn't this be experiment?

Comment (akshayabd) [reply _posts/releases/1300-09-02-early-release-beta-GA.md]: thanks - fixed

File: _posts/releases/1300-09-02-early-release-beta-GA.md [hunk 2]
```diff
@@ -1,18 +1,26 @@
...
+What is the definition for releases that are Experiments, Early Release, Beta, and GA?
 <br><br>
 ### Answer
 This is a description that varies from company to company. We decided to publish what each one of these release definitions are for Armory, and where we suggest you install them. Regardless of what stage a feature is in, we would love to hear some feedback about it.
 <br><br>
+### Experiment
+An experiment is a test version of a new feature that we think will solve some of your pain points. This is your opportunity to provide us feedback. We want to hear the good, the bad, and the ugly, so we can create the best feature to solve your problem.
+<br><br>
+**Where To Install an Experiment**
+Not production! No, really don’t do it! An experimental feature is meant to give the user access to a concept we think would be useful and gives Armory a chance to collect feedback on validating the usefulness and direction of the feature. Experiments are meant for a closed development environment or a learning environment that can be destroyed at any minute and rebuilt.
```
Comment (brian-armory) [inline _posts/releases/1300-09-02-early-release-beta-GA.md]: Need a space here

Comment (akshayabd) [reply _posts/releases/1300-09-02-early-release-beta-GA.md]: Updated

**Pull request merged**